### ISSUE TITLE
Para wireless bluetooth module 

### ISSUE THREAD

**Issue by mortehoe** [opentx/opentx#7175]
I have installed the para bluetooth module on Frsky horus X10s together with ISRM module.Wireless module which worked properly when installed with opentx 2.3.2 'internalaccess' enabled.I have since removed ISRM module (too many issues) and installed opentx without 'internalaccess' function.
Since then the bluetooth module appears to operate with less resolution (seems to step slower when servos deflecting) in bluetooth trainer mode linking with xlite pro.

**Comment by 3djc** [comment]
We reviewed the code and could not find any reason why removing internalaccess would change in any way BT operations

**Comment by kilrah** [comment]
Probably just RF interference from the internal module causing it to drop packets...

**Comment by mortehoe** [comment]
![Inked20191212_153900_LI](https://user-images.githubusercontent.com/3820551/70729462-2e74d880-1cfb-11ea-9aa2-3f090a962a09.jpg)
I have replaced the ISRM module (internalacces)and it does not seem related to the module the only thing updated since last test on 2.3 nightlies is now to 2.3.3 rc1.
This exclamation appears on initial setup of bluetooth trainer mode and I have to clear and discover again before reconnection (also after powering off/on it can take a couple of minutes to connect again.
![Inked20191212_154429_LI](https://user-images.githubusercontent.com/3820551/70729905-eefabc00-1cfb-11ea-851a-a179c8673543.jpg)
when discovering devices (on master) at first it showed both devices addresess then it only showed one as pictured above after having to re-discover (correct address)
I am fairly sure when I tested originally when  the issue of bluetooth not working (#6910) was fixed it was working fine in nightlies with no loss of resolution.
